simulation_parameters/microbe_stage/biomes.json

```diff
@@ -42,16 +42,20 @@
                     "name": "SMALL_IRON_CHUNK",
                     "meshes": [
                         {
-                            "scenePath": "res://assets/models/IronRock1.tscn"
+                            "scenePath": "res://assets/models/IronRock1.tscn",
+                            "collisionShapePath": "res://src/microbe_stage/collision_shapes/IronRock1.shape"
```

Comment:
I think that as the collision shapes are very strongly related to the models, all the .shape files should be put next to the .mesh files they are related to.

Reply:
yeah, the collision shapes fit better somewhere in the assets folder than in the src/microbe_stage folder